rest_framework/decorators.py

```diff
@@ -109,10 +108,12 @@ def decorator(func):
     return decorator
 
 
-def detail_route(methods=['get'], **kwargs):
+def detail_route(methods=None, **kwargs):
     """
     Used to mark a method on a ViewSet that should be routed for detail requests.
     """
+    methods = ['get'] if methods is None else methods
```

Comment:
Minor one, this, but I tend to prefer using brackets around the if clause for visual clarity, so I normally do...

    ['get'] if (methods is None) else methods